examples/simple/src/posts/PostShow.tsx

```diff
@@ -1,117 +1,128 @@
-                    </Tab>
-                    <Tab label="post.form.comments">
-                        <ReferenceManyField
-                            addLabel={false}
-                            reference="comments"
-                            target="post_id"
-                            sort={{ field: 'created_at', order: 'DESC' }}
-                        >
-                            <Datagrid>
-                                <DateField source="created_at" />
-                                <TextField source="author.name" />
-                                <TextField source="body" />
-                                <EditButton />
-                            </Datagrid>
-                        </ReferenceManyField>
-                        <CreateRelatedComment />
-                    </Tab>
-                </TabbedShowLayout>
-            </ShowView>
-        </ShowContextProvider>
-    );
+const PostShow = (props) => {
+  const controllerProps = useShowController(props);
+  const locale = useLocale();
+  return (
+    <ShowContextProvider value={controllerProps}>
+      <ShowView title={<PostTitle />}>
+        <TabbedShowLayout>
+          <Tab label="post.form.summary">
+            <TextField source="id" />
+            <TextField source="title" />
+            {controllerProps.record &&
+              controllerProps.record.title ===
+                "Fusce massa lorem, pulvinar a posuere ut, accumsan ac nisi" && (
+                <TextField source="teaser" />
+              )}
+            <ArrayField source="backlinks">
+              <Datagrid>
+                <DateField source="date" />
+                <UrlField source="url" />
+              </Datagrid>
+            </ArrayField>
+          </Tab>
+          <Tab label="post.form.body">
+            <RichTextField
+              source="body"
+              stripTags={false}
+              label=""
+              addLabel={false}
+            />
+          </Tab>
+          <Tab label="post.form.miscellaneous">
+            <ReferenceArrayField
+              reference="tags"
+              source="tags"
+              sort={{ field: `name.${locale}`, order: "ASC" }}
+            >
+              <SingleFieldList>
+                <ChipField source={`name.${locale}`} />
+              </SingleFieldList>
+            </ReferenceArrayField>
+            <DateField source="published_at" />
+            <SelectField
+              source="category"
+              choices={[
+                { name: "Tech", id: "tech" },
+                { name: "Lifestyle", id: "lifestyle" }
+              ]}
+            />
+            <ArrayField source="pictures">
+              <Datagrid>
+                <ImageField source="url" />
+                <ArrayField source="metas.authors">
+                  <Datagrid>
+                    <TextField source="name" />
+                  </Datagrid>
+                </ArrayField>
+              </Datagrid>
+            </ArrayField>
```

Comment:
This is what I added to the `simple` example (+ updating react-admin in package.json)